src/linux/UserInteractionsLinux.cpp

```diff
@@ -31,6 +33,13 @@ namespace Surge
 
         void openURL(const std::string &url)
         {
+           if (vfork()==0)
+           {
+              if (execlp("xdg-open", "xdg-open", url.c_str(), (char*)NULL) < 0)
```

Comment:
last argument can be `nullptr`
I would just test this in a system without xdg-open or purposefully renaming things here in order to make it call something that does not exist, and see how that behaves.
if that works, then we should be fine.